.github/workflows/unix.yml

```diff
@@ -0,0 +1,254 @@
+name: Unix build
```

Comment:
Nit: should we change the name to something other than "build"? Otherwise it is confusing what is the difference between "Unix build" and Unix build in "static builds". Maybe "Unix tests"?
"CI" was a terrible name, thanks for changing it.

Reply:
I agree, XXX tests sounds better